packages/tools/src/tools/annotation/LengthTool.ts

```diff
@@ -138,6 +138,10 @@ class LengthTool extends AnnotationTool {
     points: Types.Point3[],
     options?: {
       annotationUID?: string;
+      instance?: LengthTool;
```

Comment:
can we call this one toolInstance? since we have instance a lot in OHIF and instanceMetadata etc